python-build/action.yml

```
@@ -0,0 +1,64 @@
+name: Test and Build Package
```

yamllint (1.29.0-1)

[error] wrong new line character: expected \n

(new-lines)